### DISCUSSION TITLE
[Feature request] Allow open games in new tab/window

### DISCUSSION THREAD

**Discussion by marcop93** [ChristopherKlay/StadiaEnhanced#92] [Ideas & Requests]
Usually I like to open everything in a new tab while in a list.
The suggestion would be able to open a game, while in a list, in a new tab (something that Stadia should do by itself...)

Noticed that the full url is composed of two variables: `https://stadia.google.com/store/details/<app-id>/sku/<sku>`

In the store list (example in the printscreen, in case it's useful), both variables are in two attributes, `data-app-id` and `data-sku-id`, exactly where a link could/should be added at.

![image](https://user-images.githubusercontent.com/6834154/104382241-ae382c80-5525-11eb-977b-bc99f065925a.png)

**Comment by ChristopherKlay** [comment]
I think the main point Stadia tries to hide here is that the entire website is currently build to self-contain content and only load into itself, instead of creating instances (i.e. other tabs), most likely due to some issues with how data is retrieved.

If you open your achievements a couple of times, you even get a error msg from the server and get logged out sometimes - which is why i sadly had to remove those from the shortcuts.

This might require some testing and making sure none of those issues pop up when used too much / from certain pages, but i'll add it to the to-do list. Unless Stadia does some funky stuff when loading, this should be as simple as wrapping the entire content in a link element.

**Reply by marcop93** [reply]
Hey! 
I wasn't aware of that but really looks a strange behaviour from them. 
I manually added the wrapped the DIV in a link and it seem to work well and not breaking the page format :grin: also noticed that after a few requests (10-20) in only a few seconds it starts giving 429's but that looks acceptable looks acceptable :)

Edit: also noticed that each tab costs a bit (CPU-wise) to open, maybe that's what they want to prevent eheh
